_pytest/test_multitenancy.py

```diff
@@ -28,110 +34,136 @@ def app(component_builder):
         'port': -1,  # unused in test app
         "backend": "mitie",
         "path": os.path.join(root_dir, "test_models"),
-        "data": os.path.join(root_dir, "data/demo-restaurants.json"),
-        "server_model_dirs": {
-            "one": "test_model_mitie",
-            "two": "test_model_mitie_sklearn",
-            "three": "test_model_spacy_sklearn",
-        }
+        "data": os.path.join(root_dir, "data/demo-restaurants.json")
     }
-    config = RasaNLUConfig(cmdline_args=_config)
-    application = create_app(config, component_builder)
-    return application
+    url = '127.0.0.1'
+    port = 5000
+    train_models(component_builder)
+    pid = os.fork()
+    if pid == 0:
```

Comment:
okay - lets add this description there to make sure we remember what we did there in the future.